SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -1,137 +1,327 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace SecurityService.IntergrationTests.Common
+﻿namespace SecurityService.IntergrationTests.Common
 {
-    using System.Data.SqlClient;
-    using System.Net;
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
-    using BoDi;
     using Client;
+    using Ductus.FluentDocker;
     using Ductus.FluentDocker.Builders;
-    using Ductus.FluentDocker.Model.Builders;
+    using Ductus.FluentDocker.Commands;
+    using Ductus.FluentDocker.Common;
     using Ductus.FluentDocker.Services;
     using Ductus.FluentDocker.Services.Extensions;
-    using OpenQA.Selenium;
-    using OpenQA.Selenium.Chrome;
-    using TechTalk.SpecFlow;
+    using Shared.Logger;
 
-    public class DockerHelper
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <seealso cref="Shared.IntegrationTesting.DockerHelper" />
+    public class DockerHelper : Shared.IntegrationTesting.DockerHelper
     {
-        public INetworkService TestNetwork;
+        #region Fields
+
+        /// <summary>
+        /// The security service client
+        /// </summary>
+        public ISecurityServiceClient SecurityServiceClient;
 
-        public IContainerService SecurityServiceTestUIContainer;
+        /// <summary>
+        /// The security service container name
+        /// </summary>
+        public String SecurityServiceContainerName;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)